test/controllers/IneligibleControllerSpec.scala

```diff
@@ -62,7 +64,9 @@ class IneligibleControllerSpec extends ItSpec {
             case TaxRegime.Epaye => Some(Set(TdAll.payeEnrolment))
             case TaxRegime.Vat   => Some(Set(TdAll.vatEnrolment))
             case TaxRegime.Sa    => Some(Set(TdAll.saEnrolment))
+            case TaxRegime.Sia   => Some(Set(TdAll.saEnrolment))
```

Comment:
here you'd just set enrolments to `None` here or  `Some(Set.empty)` depending on how it's being used later on